lib/modules/manager/custom/jsonata/readme.md

```diff
@@ -0,0 +1,195 @@
+With `customManagers` using `JSONata` queries you can configure Renovate so it finds dependencies in JSON files, that are not detected by its other built-in package managers.
```

Comment:
Docs generation needs an update too, otherwise this page will not included.

Reply:
Custom manager logic is already handled in `tools/docs/maangers.ts`